dojo/settings/settings.dist.py

```diff
@@ -223,7 +223,8 @@
     DD_EDITABLE_MITIGATED_DATA=(bool, False),
     # new feature that tracks history across multiple reimports for the same test
     DD_TRACK_IMPORT_HISTORY=(bool, True),
-
+    # Delete Auditlogs older than x month; -1 to keep all logs
+    DD_AUDITLOG_FLUSH_RETENTION_PERIOD=(int, 12),
```

Comment:
Good question, what should be the default value here?

Reply:
I like the idea of using -1 as the default so there's no change in current behavior - aka people have to opt-in to this change in functionality.

Reply:
Alright I get your points. So I will switch it to -1 to keep the current behavior and users should actively change the value to enable the feature